Update docs with new workspace commands (#721)
hashicorp/terraform-ls: docs/commands.md

}
```

### `module.calls`

> List of module which are called from the current module.

Reviewer: ```suggestion
List of modules called by the module under the given URI.

Empty array may be returned when e.g.
  - the URI doesn't represent a module
  - the configuration is invalid
  - modules are not installed
  - there are no module calls

The data is sourced from the local cache within `.terraform` and so it may not necessarily represent newly added module calls in the configuration until they're installed via `get` or `init`.
```

**Arguments:**

 - `uri` - URI of the directory of the module in question, e.g. `file:///path/to/network`

**Outputs:**

 - `v` - describes version of the format; Will be used in the future to communicate format changes.
 - `module_calls` - array of modules which are called from the module in question

>    - `name` - a unique identifier for this particular module

Reviewer: ```suggestion
   - `name` - the reference name of this particular module (i.e. `network` from `module "network" { ...`)
```

>    - `source_addr` - the source address given for this module in configuration

Reviewer: ```suggestion
   - `source_addr` - the source address given for this module call (e.g. `terraform-aws-modules/eks/aws`)
```

>    - `version` - the exact version of the module

Reviewer: ```suggestion
   - `version` - version constraint of the module call; applicable to modules hosted by the Terraform Registry (e.g. `~> 1.0`
```

Reviewer: In retrospect I guess we should've called this `version_constraint`, but that's totally unrelated to this docs PR 😄

>    - `source_type` - source of the Terraform module, e.g. github or tfregistry

Reviewer: ```suggestion
   - `source_type` - source of the Terraform module, e.g. `github` or `tfregistry`
```

   - `docs_link` - a link to the module documentation; if available
   - `dependent_modules` - array of dependent modules with the same structure as `module_calls`

```json
{
  "v": 0,
  "module_calls": [
    {
      "name": "child",
      "source_addr": "./child",
      "source_type": "local",
      "dependent_modules": []
    },
    {
      "name": "vpc",
      "source_addr": "terraform-aws-modules/vpc/aws",
      "version": "3.11.0",
      "source_type": "tfregistry",
      "docs_link": "https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws/3.11.0",
      "dependent_modules": []
    }
  ]
}
```

### `module.providers`

Provides information about the providers of the current module, including requirements and
installed version.

**Arguments:**

 - `uri` - URI of the directory of the module in question, e.g. `file:///path/to/network`

**Outputs:**

 - `v` - describes version of the format; Will be used in the future to communicate format changes.
 - `provider_requirements` - map of provider FQN string to requirements object

>    - `display_name` - a user-friendly FQN string, simplified for readability

Reviewer: ```suggestion
   - `display_name` - a human-readable name of the provider (e.g. `hashicorp/aws`)
```

>    - `version_constraint` - a comma-separated list of version constraints

Reviewer: ```suggestion
   - `version_constraint` - a comma-separated list of version constraints (e.g. `>= 1.0, < 1.2`)
```

   - `docs_link` - a link to the provider documentation; if available

>  - `installed_providers` - map of provider FQN string to installed version string; can be empty if none installed

Reviewer: ```suggestion
 - `installed_providers` - map where _key_ is the provider FQN and _value_ is the installed version of the provider; can be empty if none are installed
```
